api/opds2/publication.py

```diff
@@ -165,6 +165,9 @@ def parseEditionToPublication(self, editionRecord):
         # Covers
         self.findAndAddCover(editionRecord)
 
+        #host = 'digital-research-books-beta'\
+            #if os.environ['ENVIRONMENT'] == 'production' else 'drb-qa'
+
         # Acquisition Links
         for item in editionRecord.items:
             for link in item.links:
```

Comment:
Let's move this first `addLink` call to an `else` block if we aren't presenting it in the reader. Otherwise for reader links we'll be putting that object in the OPDS object twice

Reply:
Sounds good to me.